frontends/concrete-python/concrete/fhe/tracing/tracer.py

```diff
@@ -382,6 +382,12 @@ def _trace_numpy_operation(operation: Callable, *args, **kwargs) -> "Tracer":
             dtype = kwargs.get("dtype", np.int64)
             return Tracer(Node.constant(np.zeros(args[0].shape, dtype=dtype)), [])
 
+        if operation == np.expand_dims:
+            assert len(args) in {1, 2}
```

Comment:
Is it guaranteed to be checked before ? 
If not, can't we throw the same error as the original np.expand_dims ?

Reply:
I forgot to delete this code, the one in the bottom should be enough.

Reply:
As for guarantee, it's guaranteed by numpy before it forwards the call to us:
```
Traceback (most recent call last):
  File "/home/default/Documents/Projects/Zama/concrete/frontends/concrete-python/.playground/demo.py", line 9, in <module>
    print(f(x).shape)
  File "/home/default/Documents/Projects/Zama/concrete/frontends/concrete-python/.playground/demo.py", line 7, in f
    return np.expand_dims(x, -1, 3)
  File "<__array_function__ internals>", line 198, in expand_dims
TypeError: expand_dims() takes 2 positional arguments but 3 were given
```